### ISSUE TITLE
dialog { visibility: hidden; } breaks HTML autofocus 

### ISSUE THREAD

**Issue by Gerbert-Kaandorp** [beercss/beercss#397]
I am trying to follow the HTML specification as presented by MDN
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dialog

> When a dialog opens, the browser, by default, gives focus to the first element that can be focused within the dialog. In this example, the [autofocus](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus) attribute is applied to the "Close" button, giving it focus when the dialog opens, as this is the element we expect the user will interact with immediately after the dialog opens.

I observe no autofocus when opening the dialog with javascript (calling showModal)
Even with the attribute manually provided.
```
<dialog>
<form><input placeholder="Search text" name="q" autofocus /></form>
</dialog>
```
I have traced the issue down to 
```
dialog { 
...
visibility: hidden; 
...
}
```

If I remove this line, autofocus works. 

In my initial observation nothing else broke.

Can it be removed? 😀

Cheers!

**Comment by leonardorafael** [comment]
I'm stating the tests by removing the visibility selector, but this breaks the dialog animations and all elements of beercss site become unclickable. I will try another way. Not too simple as expected 😅

**Comment by leonardorafael** [comment]
Available in [v3.8.0](https://github.com/beercss/beercss/releases/tag/v3.8.0)

**Comment by Gerbert-Kaandorp** [comment]
Amazing!

Thank you very much for your support!
